docs/doxygen/demos/demos_subpages.txt

```diff
@@ -33,11 +33,22 @@ Messages in this demo are sent at QoS 0, which guarantees at most one delivery a
 
 /**
 @page mqtt_demo_serializer Serializer Demo
-@brief Uses plaintext sockets to establish a TCP connection, and demonstrates QoS 0 implementation using lightweight serializer API.
+@brief Uses a POSIX socket based transport interface implementation to establish a TCP connection, and demonstrates QoS 0 implementation using only the MQTT serializer API.
```

Comment:
This should not mention about the transport implementation (as it does not use the MQTT API that depends on the transport interface)
```suggestion
@brief Uses a POSIX socket API to establish a TCP connection, and demonstrates QoS 0 implementation using only the MQTT serializer API.
```